app/coffee/BucketController.coffee

```diff
@@ -0,0 +1,30 @@
+settings = require("settings-sharelatex")
+logger = require("logger-sharelatex")
+FileHandler = require("./FileHandler")
+metrics = require("metrics-sharelatex")
+Errors = require('./Errors')
+
+module.exports = BucketController =
+
+	getFile: (req, res)->
+		{bucket} = req.params
+		key = req.params[0]
+		{format, style} = req.query
+		credentials = settings.filestore.s3[bucket]
```

Comment:
Can we add the settings to  settings.defaults.coffee with corresponding env variables to set them. The env might be a bit messy, everything in the k8 world is an env variable so it might have to be a JSON string we pass through then settings does a JSON.parse